lib/client-assets.php

```diff
@@ -566,7 +566,13 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	);
 
 	// Initialize the editor.
-	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() { wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost ); } );' );
+	$editor_settings = array(
+		'wideImages' => get_theme_support( 'wide-images' ),
+	);
+	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() {'
+		. 'wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost, ' . json_encode( $editor_settings ) . ' ); '
```

Comment:
Instead of moving more values to the global `wpEditorL10n` should we not opt to move more values into the object being passed into `createEditorInstance`? I think we should eliminate `wp_localize_script()` and explicitly pass what is being exported as `wpEditorL10n` into the editor here as well. There's going to be a need to export more data from PHP into the editor, such as the server's date formatting in #2013. So the more that we can establish a single pattern for getting data from PHP into JS the better.